lib/bag_courier.rb

```
@@ -124,6 +126,16 @@ def deliver
         @data_transfer.transfer(bag.data_dir)
         track!(status: "copied")
 
+
+        if @validator
+          track!(status: "validating")
+          @validator.validate
+          track!(status: "validated")
+        else
+          track!(status: "invalidated")
+          raise BagValidationError, "validation is skipped since validator is nil: bag_id=#{@bag_id}"
+        end
```

I don't think we want to raise an error here, since not providing a `validator` should be OK. I'd probably remove line 136 and change the status to `validation_skipped`?